src/Parser/Core/Modules/Spells/BFA/AzeriteTraits/SecondaryStatProcTrait.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+
+import { formatNumber, formatPercentage } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import Analyzer from 'Parser/Core/Analyzer';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'Interface/Others/TraitStatisticBox';
+import StatTracker from 'Parser/Core/Modules/StatTracker';
+
+const CRIT = 'crit';
+const VERSA = 'versa';
+const MASTERY = 'mastery';
+const HASTE = 'haste';
+
+const STATS = {
```

Comment:
bit skeptical about defining function names as strings but at this point in the code there's no `statTracker` instance yet ...